src/Microsoft.DotNet.Wpf/src/PresentationFramework/MS/Internal/Printing/printdlgexmarshaler.cs

```diff
@@ -332,31 +331,23 @@ string printerName
                     PrintQueueIndexedProperty.QueueAttributes
                 };
 
-                SystemDrawingHelper.NewDefaultPrintingPermission().Assert();  //BlessedAssert
-                try
+                //
+                // Get the PrintQueue instance for the printer
+                //
+                using (LocalPrintServer server = new LocalPrintServer())
                 {
-                    //
-                    // Get the PrintQueue instance for the printer
-                    //
-                    using (LocalPrintServer server = new LocalPrintServer())
+                    foreach (PrintQueue queue in server.GetPrintQueues(props, types))
                     {
-                        foreach (PrintQueue queue in server.GetPrintQueues(props, types))
+                        if (printerName.Equals(queue.FullName, StringComparison.OrdinalIgnoreCase))
                         {
-                            if (printerName.Equals(queue.FullName, StringComparison.OrdinalIgnoreCase))
-                            {
-                                printQueue = queue;
-                                break;
-                            }
+                            printQueue = queue;
+                            break;
                         }
                     }
-                    if (printQueue != null)
-                    {
-                        printQueue.InPartialTrust = true;
-                    }
                 }
-                finally
+                if (printQueue != null)
                 {
-                    CodeAccessPermission.RevertAssert();
+                    printQueue.InPartialTrust = true;
```

Comment:
I am not sure true is the correct value here, see: https://docs.microsoft.com/en-us/dotnet/api/system.printing.printqueue.inpartialtrust?view=netframework-4.8.

Just look into this a bit more before setting it  I know the old code did it, but that also demanded permissions, so I'm not sure about it.

Reply:
Hmm, looking at it, I wonder if this was ever really set correctly.

Reply:
We decided to leave this as is for now.  We can keep the public surface and just fixup the value post-Preview 7 if some errors arise due to it.